pom.xml

```diff
@@ -117,6 +136,23 @@
           -->
           <useSystemClassLoader>false</useSystemClassLoader>
         </configuration>
+        <dependencies>
+          <dependency>
+            <groupId>org.junit.platform</groupId>
+            <artifactId>junit-platform-surefire-provider</artifactId>
+            <version>1.2.0</version>
+          </dependency>
+          <dependency>
+            <groupId>org.junit.jupiter</groupId>
+            <artifactId>junit-jupiter-engine</artifactId>
+            <version>${junit-jupiter-version}</version>
+          </dependency>
+          <dependency>
+            <groupId>org.junit.vintage</groupId>
```

Comment:
Optional : As all the unit tests are moved to junit 5 , I don't think vintage jar is needed

Reply:
Also `<groupId>junit</groupId>` dependency can be removed